theme/icons.go

```diff
@@ -160,6 +160,16 @@ const (
 	// Since 2.0.0
 	IconNameDocumentSave fyne.ThemeIconName = "documentSave"
 
+	// IconNameDotsHorizontal is the name of theme lookup for three horizontal dots.
+	//
+	// Since 2.0.0
+	IconNameDotsHorizontal fyne.ThemeIconName = "dotsHorizontal"
```

Comment:
"Dots" seems rather literal, a theme could choose to replace "More" with an icon that is not dots...

Reply:
Dots is the MD name, but could be changed to More

Reply:
Do you mean it's the name of the file, or the name of it's usage?
Maybe there is a better name than "More" as well, perhaps someone else has thoughts.

Reply:
I agree "More" is a more semantic name for the AppTabs use case (overflow tabs), but not for DocTabs use case (all tabs)

Reply:
Yes I guess. Are there other instances that we can see the icon being used that might help us?

Reply:
I couldn't find any, renamed to "More"